packager/packager.cc

```diff
@@ -1421,36 +1419,17 @@ ast::ParsedFile rewritePackagedFile(core::Context ctx, ast::ParsedFile parsedFil
                         "of its parent directories",
                         ctx.file.data(ctx).path(), PACKAGE_FILE_NAME);
         }
-        return parsedFile;
+        return;
     }
 
     auto &pkgImpl = PackageInfoImpl::from(pkg);
 
     EnforcePackagePrefix enforcePrefix(ctx, pkgImpl, file.isPackagedTest());
-    ast::ShallowWalk::apply(ctx, enforcePrefix, parsedFile.tree);
-
-    return parsedFile;
+    ast::ConstShallowWalk::apply(ctx, enforcePrefix, tree);
 }
 
 // Re-write source files to be in packages. This is only called if no package definitions were changed.
-vector<ast::ParsedFile> rewriteFilesFast(core::GlobalState &gs, vector<ast::ParsedFile> files) {
-    Timer timeit(gs.tracer(), "packager.rewriteFilesFast");
-    for (auto &file : files) {
-        core::Context ctx(gs, core::Symbols::root(), file.file);
-        if (file.file.data(gs).isPackage()) {
-            {
-                core::MutableContext ctx(gs, core::Symbols::root(), file.file);
-                runPackageInfoFinder(ctx, file, gs.packageDB().extraPackageFilesDirectoryUnderscorePrefixes(),
-                                     gs.packageDB().extraPackageFilesDirectorySlashPrefixes());
-            }
-            // Re-write imports and exports:
-            file = validatePackage(ctx, move(file));
-        } else {
-            file = rewritePackagedFile(ctx, move(file));
-        }
-    }
-    return files;
-}
+void rewriteFilesFast(core::GlobalState &gs, absl::Span<const ast::ParsedFile> files) {}
```

Comment:
Unfortunately it looks like you wrote this comment before I realized that I had to back out these changes, because it was causing test failures. It turns out that my attempt to delete the `runPackageInfoFinder` call broke tests, because while we're not using `runPackageInfoFinder` to update the package DB, we _were_ using it to do some simple rewrites on the trees. I'm working on fixing that in a separate change.